[2025-04-09T06:58:35.496+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_lastfm manual__2025-04-09T06:58:30.412872+00:00 [queued]>
[2025-04-09T06:58:35.508+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_lastfm manual__2025-04-09T06:58:30.412872+00:00 [queued]>
[2025-04-09T06:58:35.508+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T06:58:35.593+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extract_lastfm> on 2025-04-09 06:58:30.412872+00:00
[2025-04-09T06:58:35.603+0000] {standard_task_runner.py:60} INFO - Started process 154 to run task
[2025-04-09T06:58:35.607+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'extract_lastfm', 'manual__2025-04-09T06:58:30.412872+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmpgfi0ajng']
[2025-04-09T06:58:35.609+0000] {standard_task_runner.py:88} INFO - Job 67: Subtask extract_lastfm
[2025-04-09T06:58:35.748+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.extract_lastfm manual__2025-04-09T06:58:30.412872+00:00 [running]> on host fc23817944d5
[2025-04-09T06:58:35.938+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='extract_lastfm' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T06:58:30.412872+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T06:58:30.412872+00:00'
[2025-04-09T06:58:35.943+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T06:58:35.945+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/extract_lastfm.py']
[2025-04-09T06:58:35.966+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T06:58:37.651+0000] {subprocess.py:93} INFO - ‚ùå Error al extraer Last.fm: [Errno 2] No such file or directory: '../data/processed/spotify.parquet'
[2025-04-09T06:58:37.797+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T06:58:37.864+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=extract_lastfm, execution_date=20250409T065830, start_date=20250409T065835, end_date=20250409T065837
[2025-04-09T06:58:37.946+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T06:58:38.004+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
